altair/examples/stripplot.py

```diff
@@ -0,0 +1,39 @@
+"""
+Stripplot
+-----------------
```

Comment:
This line needs to be the same length as the title above, or sphinx will complain when generating the HTML.